### PR TITLE
Mccalluc/concentration fields

### LINKED ISSUE #550: New antibodies version with concentration fields
Add a column for a concentration_value and concentration_unit. Concentration_unit should be a constrained to "ug/ml".

### PR DESCRIPTION
**Pull request by mccalluc**
@cebriggs7135 : Can you use the github suggest tool to fill in the descriptions on the yaml? Thanks

Fix #550

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/antibodies.yaml [hunk 1]
```diff
@@ -32,6 +33,18 @@ fields:
       required: False
       pattern: '1/\d+'
     example: 1/200
+  -
+    name: concentration_value
+    description: TODO!
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/antibodies.yaml:38]:
```suggestion
    description: The concentration value of the antibody preparation.
```

File: src/ingest_validation_tools/table-schemas/antibodies.yaml [hunk 2]
```diff
@@ -32,6 +33,18 @@ fields:
...
+    type: number
+    constraints:
+      required: False
+  -
+    name: concentration_unit
+    description: TODO!
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/antibodies.yaml:44]:
```suggestion
    description: The concentration units of the antibody preparation.
```

**Review by cebriggs7135 — CHANGES_REQUESTED** [review]

File: src/ingest_validation_tools/table-schemas/antibodies.yaml (hunk 2 above)
Comment (cebriggs7135) [reply src/ingest_validation_tools/table-schemas/antibodies.yaml]: @mccalluc  concentration units should be optional, as the value is.

**Review by cebriggs7135 — CHANGES_REQUESTED** [review]

File: src/ingest_validation_tools/table-schemas/antibodies.yaml (hunk 2 above)
Comment (mccalluc) [reply src/ingest_validation_tools/table-schemas/antibodies.yaml]: > @mccalluc concentration units should be optional, as the value is.

Sorry, I missed this comment. I don't believe we should do it that way: We do not have a mechanism to make one field required only if another is present. Between having a default unit in the column, when there's no value for it to apply to, or missing the unit when it is needed, I prefer the former.

**Pull request merged**